dm/dm/config/task.go

```diff
@@ -788,6 +792,19 @@ func (c *TaskConfig) adjust() error {
 			defaultCfg := DefaultSyncerConfig()
 			inst.Syncer = &defaultCfg
 		}
+		if inst.Syncer.QueueSize == 0 {
+			inst.Syncer.QueueSize = defaultQueueSize
+		}
+		if inst.Syncer.CheckpointFlushInterval == 0 {
+			inst.Syncer.CheckpointFlushInterval = defaultCheckpointFlushInterval
+		}
+		if inst.Syncer.SafeModeDuration == "" {
+			inst.Syncer.SafeModeDuration = strconv.Itoa(2 * inst.Syncer.CheckpointFlushInterval)
```

Comment:
I agree that when CheckpointFlushInterval is changed, we should align SafeModeDuration to 2 * inst.Syncer.CheckpointFlushInterval. But seems you forget to add `s` (seconds) to the value. And please check why test didn't find the problem.

And defaultSafeModeDuration can be removed